ui/app/AppLayouts/Chat/stores/CommunityTokensStore.qml

```diff
@@ -26,30 +26,39 @@ QtObject {
                            ensName: "carmen.eth",
                            walletAddress: "0xb794f5450ba39494ce839613fffba74279579268",
                            imageSource:image,
-                           amount: 3
+                           amount: 3,
+                           selfDestructAmount: 0,
+                           selfDestruct: false
                        },
                        {
                            ensName: "chris.eth",
                            walletAddress: "0xb794f5ea0ba39494ce839613fffba74279579268",
                            imageSource: image,
-                           amount: 2
+                           amount: 2,
+                           selfDestructAmount: 0,
+                           selfDestruct: false
                        },
                        {
                            ensName: "emily.eth",
                            walletAddress: "0xb794f5ea0ba39494ce839613fffba74279579268",
                            imageSource: image,
-                           amount: 2
+                           amount: 2,
+                           selfDestructAmount: 0,
+                           selfDestruct: false
                        },
                        {
                            ensName: "",
                            walletAddress: "0xb794f5ea0ba39494ce839613fffba74279579268",
                            imageSource: "",
-                           amount: 1
+                           amount: 1,
+                           selfDestructAmount: 0,
+                           selfDestruct: false
                        }
                    ])
     }
 
-    signal deployFeeUpdated(string value) // TO BE REMOVED
+    signal deployFeeUpdated(string value) // TO BE REMOVED    
+    signal selfDestructFeeUpdated(string value) // TO BE REMOVED
```

Comment:
No, value should be `int` based, not string :)

Reply:
Yeah, this is just a mocked data and first step. It will be modified in next step similar to this signal: 
`signal deployFeeUpdated(var ethCurrency, var fiatCurrency, int error)`